packages/web-components/fast-components/src/styles/patterns/button.ts

```diff
@@ -187,21 +171,17 @@ export const AccentButtonStyles = css`
         color: ${accentForegroundCutRestBehavior.var};
     }
 
-    :host([appearance="accent"]:hover) {
+    :host([appearance="accent"]:hover:not([disabled])) {
```

Comment:
I'm slightly concerned at the complexity and specificity of the selectors here. My assumption is that this is due to the fact that we're adding these styles to the base button. What if we shared the styles we're applying to the base button across each of these appearances? Would this complex selector still be needed?

Reply:
I removed the :not pseudo class. what I decided to do was add hover and active on the disabled selectors for each appearance